src/main/resources/messages.reference.conf

```diff
@@ -87,15 +87,22 @@
     },
     "h" : {
       "no players" : ${colours.negative}"No players online",
+      "cannot modify" : ${colours.negative}"You cannot modify player healths",
+      // attempted = Attempted maximum health
+      // limit = Actual limit set by configuration/server
+      "very high maximum health" : ${colours.negative}"Warning: Maximum health is > {{limit}}, this may cause FPS problems for clients"
+      "limited by server" : ${colours.negative}"Warning: Maximum health was limited by the server to {{limit}}"
       // total = how many players
       // players = list of players and their health
       "message" : ${colours.command}"Player healths ({{total}}): "${colours.secondary}"{{players}}",
+      "modify message" : ${colours.command}"New player healths ({{total}}): "${colours.secondary}"{{players}}",
       // name = the name of the player
       // percentage = 0-100 percentage health
       // health = 0-20 health
       // maxHealth = player's max health
       "player" : "{{name}} {{percentage}}%",
       "player separator" : " "
+      "modify feedback" : ${colours.command}"Your health has been modified to {{percentage}}%"
```

Comment:
Can {{percentage}} here be changed to {{health}}/{{maxHealth}}? Percentage is a bit weird in a notification and excludes any max health changes